indispensable-cosef/build.gradle.kts

```diff
@@ -39,6 +39,7 @@ kotlin {
 
 exportIosFramework(
     "IndispensableCosef",
+    transitiveExports=false,
```

Comment:
what is this about?

Reply:
removing the need to manually declare all transitive api dependencies as exports. effects multibase and encoding, mostly

Reply:
not sure i understand it, but i just assume you know what you're doing

Reply:
as you should!

Reply:
![if you say so...](https://ih1.redbubble.net/image.4058941518.6327/st,small,507x507-pad,600x600,f8f8f8.u1.jpg)